### ISSUE TITLE
ps-side unable to get it to open anything other than default

### ISSUE THREAD

**Issue by cricard7** [dpiccone/ng-pageslide#60]
Hi,

I am unable to get the ps-side (or ps-push or ps-squeeze) to work in my project. I tried first with a bootstrap page and thought that bootstrap may be interfering in some way, but tried on just a blank page copying the demo info and still no luck. Only panel I can get is a slide in panel from the right. 

This is the demo page, all I did was link the ng-pageslide directive.js to the page, otherwise everything is straight off of the demo site http://dpiccone.github.io/ng-pageslide/examples/

Any hints on how to get panels to push or even come in from a different direct properly?

Thanks in advance!

<!doctype html>
<html>

    <head>
        <meta charset='utf-8' />
        <meta http-equiv="X-UA-Compatible" content="chrome=1" />
        <meta name="description" content="Ng-pageslide : AngularJS sliding panel for serving additional content" />

        <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular.min.js"></script>
        <script src="bower_components/angular-ng-pageslide/dist/angular-pageslide-directive.min.js"></script>
        <link rel="stylesheet" type="text/css" href="http://cdn.jsdelivr.net/foundation/5.0.0/css/foundation.min.css"> 
        <title>AngularJS Pageslide directive demo</title>
        <style>
            /* Needed for hiding crollbars when pushing */
            html {
                overflow-x: hidden;
            }

            .ng-pageslide {
                background: #eee;
            }
        </style>
        <script>
            var app = angular.module("app", ["pageslide-directive"]);

            app.controller('pageslideCtrl',['$scope',function($scope){

                $scope.checked = false; // This will be binded using the ps-open attribute

                $scope.toggle = function(){
                    $scope.checked = !$scope.checked
                }

            }]);

            angular.element(document).ready(function(){

                c = angular.element(document.querySelector('#controller-demo')).scope();
            });

            // Test
            angular.element(document).ready(function () {
                if (console.assert)
                    console.assert(document.querySelectorAll('body > .ng-pageslide').length == 9, 'Made all of them')
            });
        </script>
    </head>

    <body ng-app="app">

        
        <div id="main_content_wrap" class="outer" style="padding:20px">
            <header>
            <h1>
                <a name="angularjs-pageslide-directive" class="anchor" href="#angularjs-pageslide-directive"><span class="octicon octicon-link"></span></a>AngularJS Pageslide directive
            </h1>
            </header>

            <section id="main_content" class="inner">

            <p>
                An <a href="http://angularjs.org/">AngularJS</a> directive which slides another panel over your browser to reveal an additional interaction pane.<br/>
                It does all the css manipulation needed to position your content off canvas with html attibutes and it does not depend on jQuery
            </p>
            
            <div ng-controller="pageslideCtrl">
                <a class="large button" href="" ng-click="toggle()">Open Demo</a>
                <div pageslide ps-open="checked">
                    <div style="padding:20px" id="demo-right">
                        <h2>Hello Pageslide</h2>
                        <p>Put here whatever you want</p>
                        <a ng-click="toggle()" class="button" >Close</a>
                    </div>
                </div>
            </div>
            
            <h3>Different speeds</h3>
            <div ng-controller="pageslideCtrl">
                <a class="tiny button" href="" ng-click="toggle()">Slowmo</a>
                <div pageslide ps-open="checked" ps-speed="2">
                    <div style="padding:20px" id="demo-right">
                        <h2>Hello Pageslidoo</h2>
                        <p>Put here whatever you want</p>
                        <a ng-click="toggle()" class="button" >Close</a>
                    </div>
                </div>
            </div>
            
            <div ng-controller="pageslideCtrl">
                <a class="tiny button" href="" ng-click="toggle()">Fastbro</a>
                <div pageslide ps-open="checked" ps-speed="0.25">
                    <div style="padding:20px" id="demo-right">
                        <h2>Hello Pageslidoy</h2>
                        <p>Put here whatever you want</p>
                        <a ng-click="toggle()" class="button" >Close</a>
                    </div>
                </div>
            </div>
            
            <h3>Different directions</h3>
            <div ng-controller="pageslideCtrl">
                <a class="tiny button" href="" ng-click="toggle()">Up up and above</a>
                <div pageslide ps-open="checked" ps-side="top">
                    <div style="padding:20px" id="demo-right">
                        <h2>Hello Pageslide</h2>
                        <p>Put here whatever you want</p>
                        <a ng-click="toggle()" class="button" >Close</a>
                    </div>
                </div>
            </div>
            
            <div ng-controller="pageslideCtrl">
                <a class="tiny button" href="" ng-click="toggle()">Go west</a>
                <div pageslide ps-open="checked" ps-side="left">
                    <div style="padding:20px" id="demo-right">
                        <h2>Hello Pageslide</h2>
                        <p>Put here whatever you want</p>
                        <a ng-click="toggle()" class="button" >Close</a>
                    </div>
                </div>
            </div>
            
            <h3>Push content</h3>
            <div ng-controller="pageslideCtrl">
                <a class="tiny button" href="" ng-click="toggle()">Open Sidebar</a>
                <div pageslide ps-open="checked" ps-side="left" ps-pushps-push="true" >
                    <div style="padding:20px" id="demo-right">
                        <h2>Hello Pageslide</h2>
                        <p>Put here whatever you want</p>
                        <a ng-click="toggle()" class="button" >Close</a>
                    </div>
                </div>
            </div>

            <h3>Squeeze content</h3>
            <div ng-controller="pageslideCtrl">
                <a class="tiny button" href="" ng-click="toggle()">Open Sidebar</a>
                <div pageslide ps-open="checked" ps-side="left" ps-squeeze="true" ps-cloak="false">
                    <div style="padding:20px" id="demo-right">
                        <h2>Hello Pageslide</h2>
                        <p>Put here whatever you want</p>
                        <a ng-click="toggle()" class="button" >Close</a>
                    </div>
                </div>
            </div>
            
            <br/>

            <div ng-controller="pageslideCtrl">
                <h3>As a class</h3>
                <p>Check to open: <input type="checkbox" ng-model="checked" ></p>
                <div class="pageslide" ps-open="checked" >
                    <div style="padding:20px" id="demo-ctrl">
                        <h2>Good Pageslide</h2>
                        <p>Put here whatever you want</p>
                    </div>
                </div>
            </div>

            <div ng-controller="pageslideCtrl">
                <h3>As an attribute</h3>
                <p>Check to open: <input type="checkbox" ng-model="checked" ></p> 
                <div pageslide ps-open="checked" >
                <div style="padding:20px">
                    <h2>Cool Pageslide</h2>
                    <p>Put here whatever you want</p>
                </div>
                </div> 
            </div>
            
            <div ng-controller="pageslideCtrl">
                <h3>As an element</h3>
                <p>Check to open: <input type="checkbox" ng-model="checked" ></p> 
                <pageslide ps-open="checked" >
                <div style="padding:20px">
                    <h2>Yes Pageslide</h2>
                    <p>Put here whatever you want</p>
                </div>
                </pageslide> 
            </div>
            
            

        </div>

        <script type="text/javascript">
            var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
            document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
        </script>
        <script type="text/javascript">
            try {
                var pageTracker = _gat._getTracker("UA-42627434-1");
                pageTracker._trackPageview();
            } catch(err) {}
        </script>

    </body>
</html>

**Comment by cricard7** [comment]
I tried a few different things but still can't get anything other than slide right to work(default).

Am I missing anything with the code above?

**Comment by cricard7** [comment]
Im obviously doing something incorrectly - no support here though. 
AngularStrap has some of this functionality plus much more, so I've moved to that.
